src/microbe_stage/MicrobeSpecies.cs

```diff
@@ -101,45 +102,34 @@ public void CalculateRotationSpeed()
         BaseRotationSpeed = MicrobeInternalCalculations.CalculateRotationSpeed(Organelles);
     }
 
-    public void SetInitialCompoundsForDefault()
-    {
-        InitialCompounds.Clear();
-        InitialCompounds.Add(SimulationParameters.Instance.GetCompound("atp"), 30);
-        InitialCompounds.Add(SimulationParameters.Instance.GetCompound("glucose"), 10);
-    }
-
-    public void SetInitialCompoundsForIron()
-    {
-        SetInitialCompoundsForDefault();
-        InitialCompounds.Add(SimulationParameters.Instance.GetCompound("iron"), 10);
-    }
-
-    public void SetInitialCompoundsForChemo()
-    {
-        SetInitialCompoundsForDefault();
-        InitialCompounds.Add(SimulationParameters.Instance.GetCompound("hydrogensulfide"), 10);
-    }
-
     public override void UpdateInitialCompounds()
     {
-        var simulation = SimulationParameters.Instance;
+        // Since the initial compounds are only set once per species they can't be calculated for each Biome.
+        // So, the compound balance calculation uses the default biome.
+        var biomeConditions = SimulationParameters.Instance.GetBiome("default").Conditions;
+        var compoundBalances = ProcessSystem.ComputeCompoundBalance(Organelles,
+            biomeConditions, CompoundAmountType.Current);
```

Comment:
I spotted a bug here, this needs to use the long term compound amounts not ones adjusted for current in game time for the player as that's wrong to use in this evolutionary timescale calculation. I will fix this and convert some things to constants by myself so that I can merge this now and not have to do more reviews.